ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6605,4 +6605,15 @@ VMADC:
   - mixed_air_damper_percentage_command
   - mixed_air_damper_percentage_sensor
   implements:
-  - CONTROL
\ No newline at end of file
+  - CONTROL
+
+SAQM:
```

Comment:
Compare this with your `OAQM` type in PR #1045, see my feedback there and consolidate or differentiate the two as needed.

Reply:
I intended to use OAQM for outdoor air quality monitoring, and SAQM for indoor air quality monitoring.

Reply:
I see. Instead of `SAQM`, can we add the ...pm... fields as optional to `VOCM`?